client/lib/jetpack/wpcom-upsell-controller.tsx

```diff
@@ -12,9 +13,14 @@ export default function upsellSwitch(
 		const getState = context.store.getState;
 		const siteId = getSelectedSiteId( getState() );
 		const isJetpack = isJetpackSite( getState(), siteId );
+		const isAtomic = isSiteAutomatedTransfer( getState(), siteId );
```

Comment:
I've been using `isSiteWpcomAtomic`. Do you know if there's a preference one way or the other?

Reply:
IIRC `isSiteAutomatedTransfer` works with other Atomic clients (e.g. Pressable) while `isSiteWpcomAtomic` is exclusive to WP.com. Although there shouldn't be any difference in practice now since I believe all the WP.com Atomic sites have been migrated from Pressable to the new WP.com client (a.k.a Atomic v2).